### PR TITLE
Make AddChild more generic

### PR DESCRIPTION
**Pull request by corny**

### CODE REVIEW

File: nl/nl_linux.go [hunk 1]
```diff
@@ -283,7 +283,7 @@ func NewRtAttrChild(parent *RtAttr, attrType int, data []byte) *RtAttr {
 }
 
 // AddChild adds an existing RtAttr as a child.
-func (a *RtAttr) AddChild(attr *RtAttr) {
+func (a *RtAttr) AddChild(attr NetlinkRequestData) {
```
Comment (jellonek) [inline nl/nl_linux.go]: Please update also the docstring.

Comment (corny) [reply nl/nl_linux.go]: done

**Comment by corny** [comment]
Please merge both or cherr-pick the one you like.

**Comment by corny** [comment]
I moved the second commit to my `refactoring` branch.

**Comment by corny** [comment]
Please merge this.

**Pull request merged**